lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -36,8 +43,9 @@ class LAMPApp extends Script {
 
     super(container, options);
 
-    this.databaseUser = options.databaseUser || 'lampdb';
     this.databaseName = options.databaseName || 'lampdb';
+    this.databaseUser = options.databaseUser || 'root';
```

Comment:
See line 112 below. It was not actually using this.databaseUser (lampdb) but just 'root'. so changing default to root now so that default behavior does not change.